src/pages/Login.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import './Login.css';
+import configData from '../config.json';
+import { useNavigate } from "react-router-dom";
+
+function Login() {
+  const [user, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  
+  const navigate = useNavigate();
+  
+  function handleUsernameChange(event) {
+    setUsername(event.target.value);
+  }
+
+  function handlePasswordChange(event) {
+    setPassword(event.target.value);
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    const response = await fetch(configData.LOGIN.LOGIN_URL, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({ user, password })
+    });
+    const data = await response.json();
+    console.log(data);
+    if (data.message == "Logged in"){
+      localStorage.setItem("authenticate", true)
+      navigate("/dashboard");
```

Comment:
do we have serverside sessions manager? or are we just authenticating and setting the localstorage?

Reply:
Grafana maintains the session and we are using Grafana's Login API to authenticate and then set the local storage.